core/drivers/crypto/hisilicon/hpre_main.c

```diff
@@ -0,0 +1,326 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright 2024 HiSilicon Limited.
+ * Kunpeng hardware accelerator HPRE module init.
+ */
+#include "hpre_main.h"
```

Comment:
prefer explicit include of require header file:
_io.hç, _string.h_, _sys/queue.h_, _tee_api_defines.h_, _trace.h_ (and maybe few others)

Reply:
> prefer explicit include of require header file: _io.hç, _string.h_, _sys/queue.h_, _tee_api_defines.h_, _trace.h_ (and maybe few others)

All these are included in hisi_qm.h